paper/paper.md

```diff
@@ -30,10 +30,18 @@ affiliations:
 date: 13 August 2017
 bibliography: paper.bib
 ---
+
 # Summary
+
+The NeXus scientific data format [@Koennecke:2015] is a widely adopted standard for organizing and sharing scientific data, particularly in the field of materials characterization. NeXus definitions specify the hierarchical structure and semantics of valid NeXus files and are written in XML using the NeXus Definition Language (NXDL), which itself is specified using XSD (XML Schema Definition). nyaml is a Python-based tool with both a command-line interface and an importable API that facilitates the conversion between NXDL XML and a simplified YAML representation. YAML's indentation-based syntax enhances human readability and simplifies manual editing. By providing a reliable, lossless round-trip conversion between XML and YAML, nyaml enables developers to edit NeXus definitions more efficiently without sacrificing structural or semantic fidelity.
```

Comment:
1. According to the nexus.org, neutron, x-ray, and muon science, which is completely forgotten here.
2. Here we should mention nexus file is an `HDF5` file. Actually, a nexus file is an HDF5 file storing the data according to well well-defined NeXus data model.
3. I think line breaker `(XML Schema Definition). <br> nyaml is a Python-based` would make the summary clearer.